File: engine/drift_detector.py
import sys

path, language, windows, keywords = sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4]

def print_arguments(path, language, windows, keywords):
    parameters = {
        "path": path,
        "language": language,
        "windows": windows,
        "keywords": keywords
    }
    return parameters

print(print_arguments(path, language, windows, keywords))
sys.stdout.flush()

File: gui/linkers/drift_detector.js
let {PythonShell} = require("python-shell");
const path = require("path");
const $ = require("jquery");
const swal = require("sweetalert");

function detect_drift() {
    var filePath = $("#formControlOpenFile").val();
    var language = $("#selectLanguage").val();
    var windows = $("#windowsSelect").val();
    var keywords = $("#keywordsSelect").val();

    var options = {
        scriptPath: path.join(__dirname, '/../engine/'),
        args: [filePath, language, windows, keywords]
    }

    let drift_detector = new PythonShell('drift_detector.py', options);

    drift_detector.on('message', function(message) {
        swal(message);
    });
}